website/docs/r/dns_custom_resolver_location.html.markdown

```diff
@@ -8,7 +8,11 @@ description: |-
 
 # ibm_dns_custom_resolver_location
 
-Provides a private DNS custom resolver locations resource. This allows DNS custom resolver location to create, update, and delete. For more information, about custom resolver locations, see [add-custom-resolver-location](https://cloud.ibm.com/apidocs/dns-svcs#add-custom-resolver-location).
+~> **Deprecated:**
+
+The ability to use the ibm_dns_custom_resolver_location resource to create or update Custom Resolver Location in Terraform has been removed in favor of the composite, **Custom Resolver** resource. **Please use the composite Custom Resolver resource which can handle locations as well**. For more information, check out [here](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/dns_custom_resolver#example-usage) **DO NOT mix Custom Resolver resource and Custom Resolver Location resource. Terraform will not work as expected.**
+
```

Comment:
DO NOT use Customer Resolver Location resource. If you have used Customer Resolver Location resource change it to composite Custom Resolver resource before running terraform apply.

Reply:
Done